apps/zotonic_mod_backup/src/models/m_backup_revision.erl

```diff
@@ -30,13 +30,20 @@
     get_revision/2,
     list_revisions/2,
     list_revisions_assoc/2,
+
+    periodic_cleanup/1,
+
     install/1
 ]).
 
 -include_lib("zotonic_core/include/zotonic.hrl").
 
 -define(BACKUP_TYPE_PROPS, $P).
 
+% Number of months we keep revisions
+-define(BACKUP_REVISION_RETENTION_PERIOD, 18).
```

Comment:
Maybe too long for the name, but if we add the unit, it is clear from seeing the config value what is to be expected of the value ;)
`BACKUP_REVISION_RETENTION_(IN_)MONTHS`